front/src/modules/trainschedule/components/ManageTrainSchedule/helpers/formatConfV2.ts

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import { compact } from 'lodash';
+import type { Dispatch } from 'redux';
+
+import type { TrainScheduleBase } from 'common/api/osrdEditoastApi';
+import { setFailure } from 'reducers/main';
+import type { OsrdConfState, PathStep } from 'reducers/osrdconf/types';
+import { kmhToMs } from 'utils/physics';
+
+export default function formatConfV2(
+  dispatch: Dispatch,
+  t: (arg0: string) => string,
+  osrdconf: OsrdConfState,
+  // TODO TS2 : remove this when rollingStockName will replace rollingStockId in the store
+  rollingStockName?: string,
+  ignoreTrainAddSettings = false
+) {
+  let error = false;
+  if (osrdconf.pathSteps[0] === null) {
+    error = true;
+    dispatch(
+      setFailure({
+        name: t('errorMessages.trainScheduleTitle'),
+        message: t('errorMessages.noOrigin'),
+      })
+    );
+  }
+  if (!osrdconf.departureTime) {
```

Comment:
I think we should not check departureTime but startTime right ? 
```suggestion
  if (!osrdconf.startTime) {
```

Reply:
Yep you're right.